c/testing/test_camera_c_api.c

```diff
@@ -0,0 +1,130 @@
+#include <camera_c_api.h>
+#include <engine_c_api.h>
+#include <types_c_api.h>
+#include <window_c_api.h>
+
+#include <math.h>
+#include <stdio.h>
+
+static int double_equal(double a, double b, double tolerance)
+{
+  return fabs(a - b) < tolerance;
+}
+
+static int point3_equal(const f3d_point3_t a, const f3d_point3_t b, double tolerance)
+{
+  return double_equal(a[0], b[0], tolerance) && double_equal(a[1], b[1], tolerance) &&
+    double_equal(a[2], b[2], tolerance);
+}
+
+int test_camera_c_api()
+{
+  int failed = 0;
+
+  f3d_engine_t* engine = f3d_engine_create(1);
+  if (engine == NULL)
+  {
+    printf("[ERROR] Failed to create engine\n");
```

Comment:
Let's use `puts` when there's a single string literal. It's no big deal here, but `printf` may be vulnerable sometimes (although compilers nowadays should be able to warn about it if something's wrong).
```suggestion
    puts("[ERROR] Failed to create engine");
```